components/web/ejabberd/Makefile

```diff
@@ -21,30 +21,31 @@
 
 #
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
-# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+# Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
 #
 
+BUILD_BITS= 64
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ejabberd
 COMPONENT_FMRI= web/server/$(COMPONENT_NAME)
-COMPONENT_VERSION= 17.11
-COMPONENT_REVISION= 1
+# 21.04 will fail with "ValueError: Zero padded number" during mogrify
+COMPONENT_VERSION= 21.4
 COMPONENT_PROJECT_URL=	http://www.ejabberd.im/
```

Comment:
In this case we have IPS_COMPONENT_VERSION, too. So you can use
COMPONENT_VERSION= 21.04
IPS_COMPONENT_VERSION=21.0.4
(We usually model leading zeroes like this).

Reply:
Thanks, works as it should